File: ios/RCTMGL-v10/RCTMGLLightManager.m
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(RCTMGLLightManager, RCTViewManager)

// light props
RCT_EXPORT_VIEW_PROPERTY(reactStyle, NSDictionary);

@end

File: ios/RCTMGL-v10/RCTMGLLightManager.swift
@objc(RCTMGLLightManager)
class RCTMGLLightManager: RCTViewManager {
    @objc
    override static func requiresMainQueueSetup() -> Bool {
        return true
    }
  
    override func view() -> UIView! {
      let light = RCTMGLLight()
      light.bridge = self.bridge
      return light
    }
}
